src/utils/memberListMapper.ts

```diff
@@ -0,0 +1,18 @@
+import { Member } from "@/types";
+import { localNumberFormat } from "./phoneNumberFormatter";
+import { roleFormmater } from "./memberRoleFormatter";
+
+const MEMBER_LIST_MAPPER = (member: Member) => ({
```

Comment:
Functions can't be named in this manner. Why all uppercase?

Reply:
removed